src/App.tsx

```diff
@@ -1,10 +1,44 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { error } from 'console';
 
-const USER_ID = 0;
+const USER_ID = 11050;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
```

Comment:
introduce `enum FilterType` and use it everywhere

Reply:
Not fixed